configure_infrastructure.py

```diff
@@ -0,0 +1,115 @@
+"""
+...
+"""
+import os
+import sys
+from typing import (
+    List,
+)
+
+DOCKER_WAIT = 5
+DOCKERFILE_DATASTORES = [
+    "postgres",
+    "mysql",
+    "mongodb",
+    "mssql",
+]
+EXTERNAL_DATASTORES = [
+    "snowflake",
+    "redshift",
+]
+IMAGE_NAME = "fidesops"
+
+
+def configure_infrastructure(
+    run_application: bool = False,  # Should we run the Fidesops webserver?
+    run_tests: bool = False,  # Should we run the tests after creating the infra?
+    open_shell: bool = False,  # Should we open a bash shell?
+    datastores: List[str] = [],  # Which infra should we create? If empty, we create all
+) -> None:
```

Comment:
I thought about that, but since those individual functions require the infra to be configured before anything is possible, I thought maybe this was OK. I'll see what I can do